source/Junkership.js

```diff
@@ -87,11 +98,21 @@ export default class Junkership extends Pixi.Sprite {
         this.hitBox.y = this.y + 1
     }
 
-    shoot() {
-        game.addChild(new Projectile(
-            this.x + this.width,
-            this.y + this.height/2,
-            this))
+    fire(curXpos,curYpos,curShip,wOffset,hOffset) {
+        if(this.powerUp.curPower == "peaShoota") {
```

Comment:
This would be a fantastic opportunity for an interface; a set of javascript objects that all share the same name for a method, so our junkership could call whatever instance of those objects it has.